status/reporter_gitlab_test.go

```diff
@@ -43,9 +43,10 @@ import (
 var _ = Describe("GitLabReporter", func() {
 
 	const (
-		repoUrl   = "https://gitlab.com/example/example"
-		digest    = "12a4a35ccd08194595179815e4646c3a6c08bb77"
-		projectID = "123"
+		repoUrl      = "https://gitlab.com/example/example"
+		digest       = "12a4a35ccd08194595179815e4646c3a6c08bb77"
+		projectID    = "123"
+		mergeRequest = "123"
```

Comment:
for easier debugging in case of future need, this should be different random number

Reply:
Ack, updated.

Reply:
Resolving the discussion after update.